source/reference/api/progress.txt

```diff
@@ -28,11 +28,24 @@ Request
 Response
 --------
 
-All response fields are wrapped in a top-level ``progress`` object. The
-endpoint returns either an updated status or an error.
+The ``progress`` endpoint returns etiher an updated status or an error.
+
+Successful Response
+~~~~~~~~~~~~~~~~~~~
+
+On a successful return, all response fields are wrapped in a top-level
```

Comment:
[suggestion]

For parallelism with the Error Response heading, maybe we could say: "If ``mongosync`` successfuly gets the status of the synchronization process, ..."

Reply:
Fixed